src/main/groovy/edu/oregonstate/mist/api/jsonapi/ResourceObject.groovy

```diff
@@ -1,5 +1,10 @@
 package edu.oregonstate.mist.api.jsonapi
 
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+
+@ToString
+@EqualsAndHashCode
```

Comment:
Should this be updated at the skeleton too?

Reply:
Maybe. It's definitely useful.